### DISCUSSION TITLE
It is done!

### DISCUSSION THREAD

**Discussion by munoztd0** [munoztd0/DAP_2021#26] [Announcements]
Hey everyone, @faureunige @lucilefavero @danielsgranja @ap1987-ap 

I wanted to congratulate all of you for making it to the end, and bravo for your presentations !

I know this was a really heavy project and not an easy one, and I hope it will be somehow useful for your future research.

I really enjoyed working with you and hope the best for your future. Don't hesitate to contact me if you need anything.

Best,

David

**Comment by ap1987-ap** [comment]
Hi David,

Thank you very much. Also for your support if we needed your help.
It will definitely help me for the data analyzing part of my master thesis.

Have a nice day.
Best,
Anja

**Comment by danielsgranja** [comment]
Thank you very much for all your help ! 

See you soon !

**Comment by faureunige** [comment]
Thank you very much for your guidance, this was a nice introduction to R !
